JavaScriptSPA/index.html

```diff
@@ -20,7 +20,7 @@ <h4 id="WelcomeMessage"></h4>
       // Config object to be passed to Msal on creation
       const msalConfig = {
         auth: {
-          clientId: 'Enter_the_Application_Id_here', //This is your client ID
+          clientId: 'Enter_the_Application_Id_Here', //This is your client ID
           authority: "https://login.microsoftonline.com/Enter_the_Tenant_Info_Here", //This is your tenant info
```

Comment:
What about login.microsoftonline.com, @pkanher617 ?shouln't it be tokenized?

Do I understand that the sample will no longer work out of the box?

Reply:
From what I understand, this is meant to be an AAD only sample, we have another sample for B2C here: https://github.com/Azure-Samples/active-directory-b2c-javascript-msal-singlepageapp/blob/master/index.html

This sample will run once these values have been filled in.

Reply:
@pkanher617 : I'm meaning national clouds base authority

Reply:
@jmprieur Yeah you are right, I will make this change.

Reply:
Made changes for cloud instance name

Reply:
Should it be called cloud instance name or just cloud instance? Name might confuse people to say "Arlington" Instead of login.microsoft.us

Reply:
Changed to cloud instance id.